django/requirements.txt

```diff
@@ -75,6 +75,7 @@ pypdf==5.1.0
 reportlab==4.2.0
 pyreadstat==1.2.7
 openpyxl==3.1.5
+matplotlib==3.9.2
```

Comment:
Necessary for the plots in the notebook

Reply:
Put this in requirements_dev.txt instead so it doesn't increase the prod container size.